### DISCUSSION TITLE
Is there a c++ or arduino lib for stream deck+?

### DISCUSSION THREAD

**Discussion by Till-J** [elgatosf/elgato#100] [Help]
I want to control with stream deck the motion paths of some servos driven by Teensy or Arduino directly via usb.
There is no PC or Mac in between.
Is there a lib?

**Comment by ChekTek** [comment]
Hi @Till-J, 

We don't offer any official libraries that would support connecting to Stream Deck directly via C++ at this time, but you may be able to find a third party option searching for "STREAM DECK HID Protocol". I have seen a few options out there, although I'm not sure there is one written in C++ specifically.
